frontend/src/core/state/cell.ts

```diff
@@ -22,6 +22,7 @@ export function transitionCell(
       // the cell's current code if the user modified it.
       break;
     case "running":
+      console.log("setting runstart ts to", message.timestamp);
```

Comment:
remove?

Reply:
sorry, these were debugging logs that I forgot to remove